anomaly_detector/anomaly_detector_facade.py

```diff
@@ -26,10 +28,14 @@ def run(self, single_run=False):
         self.mgr.add_steps(train)
         self.mgr.add_steps(infer)
         while break_out is False:
-            self.mgr.execute_steps()
-            print("log::facade::run")
-            time.sleep(5)
-            break_out = single_run
+            try:
+                self.mgr.execute_steps()
+                logging.info("Job ran succesfully")
+            except Exception:
```

Comment:
Can we not catch the `EmptyDataSetException` here?

Reply:
Fixed